voting_schemes/electionguard/docker/pyodide-electionguard/packages/libgmp/Makefile

```
@@ -20,7 +20,9 @@ clean:
 
 $(TARBALL):
 	[ -d $(ROOT)/downloads ] || mkdir $(ROOT)/downloads
-	wget -q -O $@ $(URL)
+	# probably the original docker with pyodide 0.16.1 has a builtin wget with old certificates and 
+	# the download fails when checking for it in the gmplib.org site
+	wget --no-check-certificate -v -O $@ $(URL)
```

Instead of relaxing the security here, we should add it to that Dockerfile: 

```diff
diff --git a/voting_schemes/electionguard/docker/ruby-node-python-electionguard/Dockerfile b/voting_schemes/electionguard/docker/ruby-node-python-electionguard/Dockerfile
index 9e3d7a0..7be3ab8 100644
--- a/voting_schemes/electionguard/docker/ruby-node-python-electionguard/Dockerfile
+++ b/voting_schemes/electionguard/docker/ruby-node-python-electionguard/Dockerfile
@@ -17,7 +17,7 @@ RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90assume-yes
 RUN gem install bundler
 
 # Install system dependencies
-RUN apt-get update --allow-releaseinfo-change && apt-get install -y sudo pipenv git vim curl
+RUN apt-get update --allow-releaseinfo-change && apt-get install -y sudo pipenv git vim curl wget
 
 # Install python and poetry
 RUN git clone https://github.com/pyenv/pyenv.git ~/.pyenv && \
```


```suggestion
	wget -v -O $@ $(URL)
```

I've just tested the local build of the docker image and it worked fine with these changes

---

good idea, done!